src/com/company/zadania/obiektowe/zadanie1/Zamowienie.java

```diff
@@ -22,22 +25,50 @@ public void dodajPozycje(Pozycja p) {
 
```

Comment:
tam wyzej powinna byc zmiana w dodajPozycje wg zadania: 'zmodyfikować metodę void dodajPozycje(Pozycja p) '

for pozcyja[i]
 if p.nazwaTowaru .eqlausl(pozycja[i].nazwatowaru) :
   pozycja[i].ilesztuk + 1
    return

pozycja[ileddodanych++] = p

Reply:
ok dzieki, nie znałem tej opcji z return